src/test/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDepositSpec.scala

```diff
@@ -150,7 +148,7 @@ class AddDatasetMetadataToDepositSpec extends UnitSpec with BeforeAndAfterAll {
   "createDcmiMetadata" should "return the expected spatial elements" in {
     val dataset = new Dataset() +=
       "DCT_SPATIAL" -> List("here", "there", "", "") +=
-      "DCX_SPATIAL_SCHEME" -> List("degrees", "degrees", "", "") +=
+      "DCX_SPATIAL_SCHEME" -> List("degrees", "RD", "", "") +=
```

Comment:
Just an extra test to see that both values for `DCX_SPATIAL_SCHEME` work properly.